.github/workflows/integrationTest.yml

```diff
@@ -144,13 +147,15 @@ jobs:
           echo "::set-output name=ec2_gpu_matrix::$(echo $(cat generator/resources/ec2_gpu_complete_test_matrix.json))"
```

Comment:
**nitpick**: `::set-output` is deprecated. [docs](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/)

Reply:
yeah we've been having this warning forever. probably have to make a sim to address all warnings. There's another one complaining about node 12.x or something

Reply:
the node 12.x thing might be fixed by upgrading checkout to v3

Reply:
Not worth blocking on this PR.